### PR TITLE
Remove cortex-m-udf as udf instruction is now part of cortex-m.

### PR DESCRIPTION
**Pull request by matoushybl**

### CODE REVIEW

**Comment by matoushybl** [comment]
Using the crate with cortex-m caused linking error.

File: async-embedded/Cargo.toml [hunk 1]
```diff
@@ -13,8 +13,7 @@ pin-utils = "0.1.0"
 typenum = "1.12.0"
 
 [target.'cfg(target_arch = "arm")'.dependencies]
-cortex-m = "0.6.2"
-cortex-m-udf = { path = "../cortex-m-udf" }
+cortex-m = "0.6"
```
Comment (Disasm) [inline async-embedded/Cargo.toml]:
```suggestion
cortex-m = "0.6.3"
```
`udf` arrived only in v0.6.3, any other version will not work.

**Review by Disasm — CHANGES_REQUESTED** [review]
Also `panic-udf` crate still refers to `cortex-m-udf`, which is wrong.

File: async-embedded/Cargo.toml (hunk 1 above)
Comment (matoushybl) [reply async-embedded/Cargo.toml]: Fixed, thanks!

**Review by Disasm — APPROVED** [review]
Thank you!
I checked this PR on a few examples and everything is ok.

**Pull request merged**